app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -101,7 +101,7 @@ def create
     end
 
     return render status: :unprocessable_entity, json: { error: "Invalid group type" } unless create_supported_groups.include?(group.group_type)
-    return head :unauthorized unless current_user.has_permission?(:can_edit_groups, group_id)
+    return head :unauthorized unless current_user.has_permission?(:can_edit_groups, group_id.to_i)
```

Comment:
I'm not sure if this change is necessary. But I don't know what is the alternative of fixing this.